facilities.csv

```diff
@@ -657,7 +663,7 @@ http://irail.be/stations/NMBS/008833274,Zichem,Ernest Claesstraat Zn,3271,Zichem
 http://irail.be/stations/NMBS/008892635,Zingem,Stationsstraat 1,9750,Zingem,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008832250,Zolder,Stationsstraat Z/n,3550,Heusden - Zolder,1,0,1,0,0,0,0,0,0,0,0,2,0,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008832334,Zonhoven,Engstegenseweg,3520,Zonhoven,1,0,1,0,0,0,0,0,0,0,0,3,0,0,0,0,0,,,,,,,,,,,,,,
-http://irail.be/stations/NMBS/008895208,Zottegem,Stationsplein 12,9620,Zottegem,1,1,1,0,1,1,1,0,0,1,1,2,0,0,0,0,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,07:15,14:30,07:15,14:30
+http://irail.be/stations/NMBS/008895208,Zottegem,Stationsplein 12,9620,Zottegem,1,0,1,0,1,1,1,0,0,1,1,2,0,0,0,0,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,07:15,14:30,07:15,14:30
 http://irail.be/stations/NMBS/008891611,Zwankendamme,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008894821,Zwijndrecht,Fortlaan 1,2070,Zwijndrecht,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008883212,Écaussinnes,Avenue De La DÉportation,7190,Ecaussinnes,1,0,1,0,1,0,1,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,
```

Comment:
Déportation?

Reply:
Nice catch, will fix, and as this file is created programmatically the PHP script will need fixing as well